### DISCUSSION TITLE
New stage 2 project-factory and old SAs

### DISCUSSION THREAD

**Discussion by karpok78** [GoogleCloudPlatform/cloud-foundation-fabric#2547] [Q&A] [answered]
Hello,

while switching to the new stage-2-PF on my organization and checking the resulting IAM bindings, I noticed that the old stage-3 PF resources remained in the code (SAs, iac buckets, IAM bindings,...).
Is the cleanup job still to come, or were they kept in preparation of a future backed-on per-environment factories ?

by the way, the overall IAM.md files seems not to reflect the latest change in FAST IAM bindings.

**Comment by ludoo** [comment, marked as answer]
The 3- project factory resources should have been removed as they are no more present in code.

Good point on the IAM file, we'll update it shortly. Tip: you can use `tools/state_iam.py`  to generate your own IAM.md file. :)

**Comment by karpok78** [comment]
My bad, there were a couple of old files that passed through the garbage collector.
Good to know about the tools/state_iam.py script
